README.org

```diff
@@ -61,6 +61,15 @@ null device
 $ open tests/current/summary.png
 #+END_SRC
 
+   You can also run basho_bench as an included app rather than an escript. In order to compile and run basho_bench as an included app, you can use =make run=. You can also include basho_bench as a dependency in another app. basho_bench can be started with =basho_bench:start()= or from the command-line with =erl -s basho_bench=. Once you have a shell with basho_bench running, you can run benchmarks with the following:
```

Comment:
Noticed earlier reference in "QuickStart" to running with Erlang R13B03 as minimum requirement but rebar.config specifies R16 or R17. Should maybe tweak that to agree with the later minimum version ? 

It might make sense to add a different header for this section, something like "** Running as an Application" since the earlier section is a once-through "quickstart".